opt_out_http_api/api_methods.py

```diff
@@ -0,0 +1,55 @@
+import json
+
+from klein import Klein
+
+
+class OptOutNotFound(Exception):
+    """ Raised when no opt out is found. """
+
+
+class API(object):
+    app = Klein()
+
+    def __init__(self):
+        self._optouts = [
+            {"id": "2468", "address_type": "msisdn", "address": "+273121100"},
+            {"id": "1234", "address_type": "facebook", "address": "fb-app"},
+            {"id": "5678", "address_type": "twitter",
+             "address": "@twitter_handle"}
+        ]
+
+# Get Opt Out Address
+
+    def get_opt_out(self, addresstype, address):
+        opt_outs = [
+            o for o in self._optouts
+            if o["address_type"] == addresstype and o["address"] == address
+        ]
+        if opt_outs:
+            return opt_outs[0]
+        return None
+
+    @app.handle_errors(OptOutNotFound)
+    def opt_out_not_found(self, request, failure):
+        request.setResponseCode(404)
+        request.setHeader('Content-Type', 'application/json')
+        return json.dumps(None)
```

Comment:
We should return a nice error message (formatted as JSON) here. Perhaps something like:
```
{
   "status": 404,
   "error": "Opt out not found.",
}
```